stacker_blueprints/firehose/base.py

```diff
@@ -233,27 +229,60 @@ class Firehose(Blueprint):
                            "Default: true",
             "default": True,
         },
-        "KeyUseArns": {
+        "KeyUserArns": {
```

Comment:
Did you mean to rename this?  Also, lost the description (same with the other `Key*Arns` below.